include/grpcpp/server_impl.h

```diff
@@ -306,7 +303,7 @@ class Server : public grpc::ServerInterface, private grpc::GrpcLibraryCodegen {
       std::unique_ptr<grpc::experimental::ServerInterceptorFactoryInterface>>
       interceptor_creators_;
 
-  const int max_receive_message_size_;
+  int max_receive_message_size_;
```

Comment:
Why is this member even needed?  It looks like the only thing it ever gets used for is populating a similar member inside of `grpc::internal::Call`, but I don't see any code in that class that actually uses that member for anything ether.  Maybe we can just remove all of this plumbing?

Reply:
It is exposed to the streaming unary sync APIs. See b/146233230 for more issues there...

Reply:
Ah, I missed that.  Okay, this makes sense, then.